src/containers/Departments/Details/styles.js

```diff
@@ -243,44 +246,46 @@ export const TeamTimelineContainer = styled.div(
       flexDirection: "column",
       justifyContent: "space-between",
       "& h2": {
-        marginBottom: "36px",
+        marginBottom: `${spacing * 18}px`,
       },
 
       [`@media screen and (max-width: ${sm}px)`]: {
         width: "100%",
         "& h2": {
-          marginBottom: "26px",
+          marginBottom: `${spacing * 13}px`,
         },
       },
     };
   }
 );
 export const TeamUserListWrapper = styled.div`
   background-color: #f05b25;
-  margin-bottom: 50px;
+  margin-bottom: ${({ theme: { spacing } }) => spacing * 25};
 `;
 
-export const SpacerContainer = styled.div(({ theme: { breakPoints } }) => {
-  const sm = breakPoints.sm;
+export const SpacerContainer = styled.div(
+  ({ theme: { breakPoints, spacing } }) => {
+    const sm = breakPoints.sm;
 
-  return {
-    height: "38px",
+    return {
+      height: `${spacing * 19}px`,
 
-    [`@media screen and (max-width: ${sm}px)`]: {
-      height: "28px",
-    },
-  };
-});
+      [`@media screen and (max-width: ${sm}px)`]: {
+        height: `${spacing * 14}px`,
+      },
+    };
+  }
+);
 
-export const DescWrapper = styled.div(({ theme: { breakPoints } }) => {
+export const DescWrapper = styled.div(({ theme: { breakPoints, spacing } }) => {
   const sm = breakPoints.sm;
 
   return {
     display: "flex",
     margin: "20px 0",
```

Comment:
use theme spacing

Reply:
Added spacing in theme and set value 2 and used it in the app.